docs/deploy.md

```diff
@@ -0,0 +1,222 @@
+# Deploying Authorino
+
+1. [Clone the repo](#clone-the-repo)
+2. [Meet the requirements](#meet-the-requirements)
+3. [Deploy](#deploy)
+   - [Option A: Local cluster](#option-a-local-cluster)
+   - [Option B: Custom deployment](#option-b-custom-deployment)
+
+## Clone the repo
+
+This is the easiest way to get Authorino's Kubernetes resources and deployment utils.
+
+```sh
+git clone git@github.com:3scale-labs/authorino.git && cd authorino
+```
+
+## Meet the requirements
+
+Having a [Kubernetes](https://kubernetes.io/) server up and running is the only actual requirement to deploy Authorino.
+
+With the tools provided in the Authorino repo nevertheless, you can easily have a local cluster setup (using [Kind](https://kind.sigs.k8s.io)). In this case, the only requirement is [Docker](https://docker.com).
+
+### Requirements: the untold truth
+
+Apart from the obvious requirement of having a running instance of a Kubernetes server, Authorino also relies on some other components and capabilities to perform its function:
+
+<!-- TODO: Add minimum required Kubernetes version -->
+
+- Permission over the Kubernetes server to create Custom Resource Definitions (CRDs). This is actually a one-time requisite, necessary only to install Authorino, and assumed to be known by cluster administrators who should be able to handle as desired through the Kubernetes API Role-Based Access Control bindings.
+- [Envoy](https://www.envoyproxy.io) proxy (or, technically, any proxy that implements the client-side of the [gRPC protocol](https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/ext_authz). With Envoy, ultimately, virtual hosts will be associated to the upstream APIs to protect, and an [External Authorization Filter](https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/ext_authz) should point to Authorino. Check [this example](/examples/envoy/envoy-deploy.yaml) if you need to.
```

Comment:
I would add this paragraph to the first part too and then rename the subtitle as suggested.